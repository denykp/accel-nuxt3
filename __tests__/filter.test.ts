import { getFlyingSuperHeroes } from "../utils/filter";

test("should return super heroes that can fly", () => {
  const flyingSuperHeroes = getFlyingSuperHeroes();

  expect(flyingSuperHeroes).toMatchSnapshot();
});
