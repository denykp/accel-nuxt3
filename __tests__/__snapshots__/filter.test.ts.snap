// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return super heroes that can fly 1`] = `
[
  {
    "name": "Jack",
    "powers": [
      "shapeshifting",
      "fly",
    ],
  },
]
`;
